src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehagebarnService.kt

```diff
@@ -74,14 +53,34 @@ class BarnehagebarnService(
 
     private fun getCorrectSortBy(sortBy: String): String =
         when (sortBy.lowercase()) {
-            "endrettidspunkt" -> "endret_tid"
+            "endrettidspunkt" -> "endretTid"
             "kommunenavn" -> "kommune_navn"
             "kommunenr" -> "kommune_nr"
             "antalltimeribarnehage" -> "antall_timer_i_barnehage"
             else -> sortBy
         }
 
-    // Hvis barnehagebarnet tilhører en annen periode eller kommer fra en annen liste ansees det som en ny melding, kunne muligens brukt barnehagebarn.id i stedet
+    private fun mapBarnehagebarnTilBarnehagebarnDto(
+        barnehagebarn: BarnehagebarnForListe,
+    ): BarnehagebarnVisningDto {
+        val fagsakIdTilFagsakStatus = fagsakRepository.finnFagsakIdOgStatusMedAktivBehandlingForIdent(barnehagebarn.getIdent()).firstOrNull()
```

Comment:
Henter dette her siden vi ikke har behov for å sortere på fagsak på noe vis og jeg ikke ønsker å legge til noe mer i SQL-en i barnehageRepository